app/models/in_progress_form.rb

```diff
@@ -91,6 +92,28 @@ def expires_after
       end
   end
 
+  def log_cfi_metric(params)
+    if form_id == '21-526EZ' && metadata['cfiMetric'].blank?
+      fd = params[:form_data] || params[:form_Data]
+      fd = JSON.parse(fd) if fd.is_a?(String)
+      if fd.present? &&
+         (fd['rated_disabilities'].present? || fd['ratedDisabilities'].present?) &&
+         (fd.dig('view:claim_type', 'view:claiming_increase') || fd.dig('view:claimType', 'view:claimingIncrease'))
+
+        max_cfi_enabled = Flipper.enabled?(:disability_526_maximum_rating) ? 'on' : 'off'
+
+        rated_disabilities = fd['rated_disabilities'] || fd['ratedDisabilities']
+        rated_disabilities.each do |dis|
+          if dis['maximum_rating_percentage'].present? || dis['maximumRatingPercentage'].present?
```

Comment:
@nanotone do we also wanna add a check to see if the `maximum_rating_percentage` matches the disabilities `rating`?